gravitee-apim-console-webui/src/management/api/history-v4/api-history-v4.component.scss

```diff
@@ -0,0 +1,22 @@
+@use 'sass:map';
+@use '@angular/material' as mat;
+@use '@gravitee/ui-particles-angular' as gio;
+@use '../../../scss/gio-layout' as gio-layout;
+
+$typography: map.get(gio.$mat-theme, typography);
+
+:host {
+  @include gio-layout.gio-responsive-content-container;
+}
+
+.history {
+  padding: 24px;
```

Comment:
i think ou can keep default `mat-card-content` padding. 
Later I'd like it to be defined by default for all the application. So that we don't have to define it page by page.

Reply:
done